### ISSUE TITLE
`CLDF_ID` in `datasets.tsv` clashes with default `cldf_id` for createdb/SQLite conversion

### ISSUE THREAD

**Issue by chrzyki** [cldf-datasets/clld_meta#5]

**Comment by xrotwang** [comment]
Ah, right. SQL being case insensitive is biting us here ...

**Comment by johenglisch** [comment]
SQL aside, I'm wondering whether I should just drop the `CLDF_ID` column.

 1. There's no guarantee that a `rdf:ID` is present in a data set at all.
 2. There's no guarantee that a `rdf:ID` is unique across data sets.
 3. There's no guarantee that `rdf:ID`'s are different in data sets within the same Zenodo record.  In fact, by default *cldfbench* just plasters its ID onto all data sets it touches, so data sets within the same record are highly likely to share the same id (e.g. in *lexibank-analysed*).

So, basically the `rdf:ID` doesn't seem to actually *identify* anything, neither to the outside nor on the inside.  At best, it doesn't really add anything -- at worst, it confuses people because of how ‘official’ the name `CLDF_ID` sounds.

**Comment by xrotwang** [comment]
Yes, I agree - also that `rdf:ID` is probably a misnomer for this property. The semantics is basically just "if you need something to distinguish two datasets, that's what you could use, in particular, if it must be ASCII-only lowercase, maybe ..."
